src/Parser.php

```diff
@@ -142,7 +142,7 @@ public function __construct() {
             [TokenKind::ArrayKeyword, TokenKind::CallableKeyword, TokenKind::BoolReservedWord,
             TokenKind::FloatReservedWord, TokenKind::IntReservedWord, TokenKind::StringReservedWord,
             TokenKind::ObjectReservedWord, TokenKind::NullReservedWord, TokenKind::FalseReservedWord,
-            TokenKind::IterableKeyword]; // TODO update spec
+            TokenKind::IterableKeyword, TokenKind::MixedReservedWord]; // TODO update spec
```

Comment:
Unrelated to this PR but I think that `IterableKeyword` should be `Reserved` or?

Reply:
Yeah, it should have been IterableReservedword like ObjectReservedWord - it isn't a separate keyword like T_ARRAY.

Since they've already created a release, you could add an `/** @deprecated */ const IterableKeyword = self::IterableReservedWord` in TokenKind

Reply:
ok, updated